docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+
+# Using The Pelican Client
+It is recommended that you start with our [Quick Start Guide](./user-quickstart-guide.mdx) where you can learn the basics of Pelican client commands and learn some useful terminology.
+
+## Before Starting
+
+### Assumptions
+
+Before using the Pelican client to interact with objects from your federation, this guide makes several assumptions:
+
+- You are on a computer where you have access to a terminal. The Pelican client is a command line tool.
+- You've already installed the version of Pelican appropriate for your system, and Pelican is accessible via your path. To test this on Linux, you can run
+```bash
+which pelican
+```
+which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](./install.mdx) to acquire a working installation.
+- You are roughly familiar with Pelican client commands and terminology as demonstrated with the [Getting Started: Accessing Data](./getting_started_access_data.mdx) page.
+
+
+### Useful Terminology
+
+**Notes on Federations:**
+Below are some notes on federations on top of their terminology. Here is the example URL we used in the [Getting Started: Accessing Data](./getting_started_access_data.mdx) page:
+```console
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+**Note:** All object paths in a federation begin with a preceding `/`, and no relative paths are allowed.
+
+**Tokens and JWT:**
+Some namespace prefixes are public, like `/ospool/uc-shared/public`, while others are protected (i.e. they require authorization). Objects in public namespaces can be downloaded by anybody, but downloading objects from protected namespaces requires you prove to the origin supporting that namespace that you are allowed to access the object. In Pelican, this is done using signed JSON Web Tokens, or *JWT*s for short. In many cases, these tokens can be generated automatically.
+
+## The Different Pelican URL Schemes
+When running client commands with a request URL, there are three different URL schemes a user can use: `pelican`, `osdf`, and `stash`. This page uses examples with the `pelican://` URL scheme. Below is a description of the three URL schemes and what to expect when they are used:
+
+### The `pelican://` URL Scheme:
+When using the `pelican://` scheme, pelican does not know what federation to communicate with and therefore, the user must specify their federation URL either as the host-name in the URL itself or within the `-f` flag. The examples below show how you can format your request URLs in both of these ways.
```

Comment:
This is our first chance to introduce the powerful concept of `pelican://` urls. I think we should avoid mentioning the `-f` flag here, because if the user is using a `pelican://` url, we want them to be including the federation hostname.

This may not be the place to do so, but we should also explain what protocol this scheme entails. Something like:

"URLs that begin with `pelican://` allow you to interact with objects from any federation simply by knowing the hostname of the federation's discovery url and the name of the object in that federation. These URLs take the form 
```
pelican://<fed discovery URL>/<path to object>
```
When a Pelican client encounters a Pelican URL, it uses the hostname in the federation discovery portion of the URL to find other services in the specified federation that will help the client discover the object's location."